src/ApiClient.js

```diff
@@ -47,7 +63,7 @@ export default class ApiClient {
 				return responseJSON;
 			})
 			.catch(err => {
-				err = err.replace(/</g, '').replace(/>/g, '');
+				err = err.toString().replace(/</g, '').replace(/>/g, '');
```

Comment:
Oh how much I would love this to be fixed, but alas this change affects a lot of other code. The issue is there are a number of places where the ApiClient methods need to throw errors for the code to work correctly. For example,  Home.js line 26 uses a try catch around the ApiClient that breaks if an error is not thrown. I think the solution to this is just not have the ApiClient doing the try catch, and let the error fall through.

Reply:
Alright. Doing it without the .toString() breaks the code locally for me which is why I put it in. Will get rid of it for the official commit